pkgs/servers/monitoring/nagios-plugins/openbsd_snmp3_check/default.nix

```
@@ -0,0 +1,42 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  openbsd_snmp3_check,
+  python3Packages,
+  stdenv,
+  testers,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "openbsd_snmp3_check";
+  version = "0.55";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "alexander-naumov";
+    repo = "openbsd_snmp3_check";
+    rev = "v${version}";
+    hash = "sha256-qDYANMvQU72f9wz8os7S1PfBH08AAqhtWLHVuSmkub4=";
+  };
+
+  installPhase = ''
```

```suggestion
  postInstall = ''
```

If we use `installPhase`, we should add `runHook preInstall` at the start and `runHook postInstall` at the end, to let downstream users who may want to add a `postInstall` by overriding the derivation (https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases).

If we use `postInstall` we don't need to.

---

I missed that from your original comment. That explanation is really helpful, thanks. I've made this change.